sacrebleu/utils.py

```diff
@@ -27,6 +26,194 @@
+            else:
+                # Already formatted in non paired-test mode
+                _str = result
+
+            new_vals.append(_str)
+
+        if is_bootstrap:
+            # Change titles
+            metric += ' / μ / ± 95% CI'
+
+        new_scores[metric] = new_vals
+
+    return new_scores
+
+
+def print_results_table(results: dict, signatures: dict, args: Namespace):
+    """Prints out a nicely formatted table for multi-system evaluation mode."""
+
+    tablefmt = args.format
+    if tablefmt in ('text', 'json'):
+        # Fallback to simple table if json is given
+        tablefmt = 'grid'
+    elif tablefmt == 'latex':
+        # Use booktabs
+        tablefmt = 'latex_booktabs'
+
+    # If paired testing has been given, this'll format the score lines
+    results = _format_score_lines(results, args.width, multiline=tablefmt == 'grid')
+
+    new_dict = {}
+
+    # Color the column names and the baseline system name and scores
+    has_baseline = False
+    baseline_name = ''
+    for name in results.keys():
+        val = results[name]
+        if val[0].startswith('Baseline:') or has_baseline:
+            if val[0].startswith('Baseline:'):
+                baseline_name = val[0]
+            has_baseline = True
+            val[0] = Color.format(val[0], 'yellow')
+        new_dict[Color.format(name, 'cyan')] = results[name]
+
+    # Finally tabulate
+    table = tabulate(
+        new_dict, headers='keys', tablefmt=tablefmt,
+        colalign=('right', ),
+        stralign='center',
+        numalign='center',
+        floatfmt=f'.{args.width}f')
+
+    print(table)
+    print()
+
+    if args.paired:
+        test_type = 'bootstrap resampling' if args.paired == 'bs' \
+            else 'approximate randomization'
+        test_sample_type = 'resampling trials' if args.paired == 'bs' else 'trials'
+        msg = f'Paired {test_type} test with {args.paired_n} {test_sample_type}'
+        bline = Color.format('baseline', 'yellow')
+        bline_name = Color.format(baseline_name, 'yellow')
+        null_hyp = Color.format('Null hypothesis', 'green')
+        pval_color = Color.format('highlighted in red', 'red')
+
+        # Print fancy header
+        print('-' * len(msg) + '\n' + msg + '\n' + '-' * len(msg))
+        print(f' - Each system is pairwise compared to {bline_name}.')
+        if args.paired == 'bs' or args.paired_ar_confidence_n != -1:
+            print('   Actual system score / estimated true mean / 95% CI are provided for each metric.')
+        else:
+            print('   Actual system score is provided for each metric.')
+        print()
+        print(f' - {null_hyp}: the system and the {bline} translations are essentially')
+        print('   generated by the same underlying process. The p-value is roughly the probability')
+        print('   of the absolute score difference (delta) between a system and the {bline} occurring due to chance.')
```

Comment:
Strictly speaking "of the absolute score difference (delta) *or higher*" and "given the null hypothesis",
but the explanatory comment is long enough so this suggestion can be ignored.

Reply:
I'm willing to make this better since I basically looked into different sources to come up with this.
So that would be:
` - Given the null hyp: ....` for the second suggestion right?

Reply:
Yes, p-value is the probability (of...) under the assumption that the null hypothesis is correct